setup.py

```diff
@@ -2,13 +2,14 @@
 
 from setuptools import setup
 
-setup(name='ffx',
-      install_requires=['gmpy','pycrypto'],
-      test_suite = 'ffx.tests',
-      version='0.0.1',
-      description='FFX',
-      author='Kevin P. Dyer',
-      author_email='kpdyer@gmail.com',
-      url='https://github.com/kpdyer/libffx',
-      packages=['ffx'],
-      )
+setup(
+    name='ffx',
+    install_requires=['gmpy', 'pycrypto', 'six'],
```

Comment:
You might want to use `pycryptodome` instead, which is a (mostly) drop replacement to `pycrypto`, which is now defunct.

Reply:
Thank you! I've changed to `pycryptodome` because it is better.